bundles/tools.vitruv.applications.umljava/META-INF/MANIFEST.MF

```diff
@@ -7,12 +7,21 @@ Bundle-Version: 3.0.1.qualifier
 Bundle-Vendor: vitruv.tools
 Bundle-RequiredExecutionEnvironment: JavaSE-17
 Require-Bundle: org.eclipse.uml2.uml;visibility:=reexport,
- org.emftext.language.java;visibility:=reexport,
  edu.kit.ipd.sdq.activextendannotations,
  edu.kit.ipd.sdq.commons.util.emf,
  edu.kit.ipd.sdq.commons.util.java,
  tools.vitruv.framework.applications,
  tools.vitruv.dsls.reactions.runtime,
- tools.vitruv.applications.util.temporary;visibility:=reexport
+ tools.vitruv.applications.util.temporary;visibility:=reexport,
```

Comment:
Are all the new packages needed?

Reply:
Probably not. I added all of them in the beginning because I didn't know which bundles I would need for the migration and forgot about removing the unnecessary ones. I'm checking the projects right now to figure out which project need what bundles.

Reply:
Done.